src/contracts/BMath.sol

```diff
@@ -130,16 +149,22 @@ contract BMath is BBronze, BConst, BNum {
   }
 
   /**
-   *
-   *    calcSingleInGivenPoolOut
-   *    tAi = tokenAmountIn              //(pS + pAo)\     /    1    \\
-   *    pS = poolSupply                 || ---------  | ^ | --------- || * bI - bI
-   *    pAo = poolAmountOut              \\    pS    /     \(wI / tW)//
-   *    bI = balanceIn          tAi =  --------------------------------------------
-   *    wI = weightIn                              /      wI  \
-   *    tW = totalWeight                          |  1 - ----  |  * sF
-   *    sF = swapFee                               \      tW  /
-   *
+   * @notice Calculate the amount of token in
+   * that should be sent for a given amount of pool tokens out when joining a pool
```

Comment:
this end of line is weird

Reply:
calculate the amount of a given token that should be sent to generate a given amount of pool tokens?